documentation/src/main/asciidoc/topics/ref_discovery_jdbc.adoc

```diff
@@ -36,16 +36,10 @@ Add a managed datasource to a {brandname} Server and use it to provide database
 include::xml/server_datasources.xml[]
 ----
 +
-. Create a JGroups stack which uses the `JDBC_PING` protocol for discovery.
+. Create a JGroups stack which uses the `JDBC_PING2` protocol for discovery.
 . Configure cluster transport to use the datasource by specifying the name of the datasource with the `server:data-source` attribute.
 +
 [source,xml,options="nowrap",subs=attributes+]
 ----
 include::xml/server-transport-datasource.xml[]
 ----
-
-
-[role="_additional-resources"]
-.Additional resources
```

Comment:
We should link to the JDBC_PING2 docs here: http://www.jgroups.org/manual5/index.html#JDBC_PING2

Reply:
Thanks, @ryanemerson, updated